model/patch/patch.go

```diff
@@ -25,6 +25,7 @@ import (
 
 // SizeLimit is a hard limit on patch size.
 const SizeLimit = 1024 * 1024 * 100
+const backportFmtString = "Backport: %s (%s)"
```

Comment:
Is this part of the commit message validations?

Reply:
I was looking to make a standardized patch description, similar to what I did for the commit queue, but it's not checked by anything.